components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -371,6 +372,12 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
               Update Preferences
             </Button>
           </div>
+          <div className="flex flex-col gap-3 order-first md:order-last">
+            <label className="text-2xl font-normal text-light-slate-11">Upgrade Access</label>
+            <div className="text-center p-3 border border-dashed rounded-xl border-light-slate-8">
```

Comment:
I don't think it needs a border around the box, and if the button can be left aligned to be consistent with the other ones.

Reply:
Sure.

I will go on and refactor the `StripeCheckoutButton` component to be more customisable for usage in this context. Its current set-up is hard coded to work this way as integrated with the Dashboard's `Reports` component.

Reply:
Making the component more flexible would be ideal in this case.

Reply:
Done 🍕